_inc/client/components/upgrade-notice-content/index.jsx

```diff
@@ -47,212 +40,29 @@ const UpgradeNoticeContent = moduleSettingsForm(
-			}
-
 			return (
 				<div className="jp-upgrade-notice__content">
 					<p>
-						{ __( 'This release of Jetpack brings with it a major new free feature!' ) }
+						{ __( 'The features you rely on, adapted for the new WordPress editor.' ) }<br />
+						{ __( 'A new editor? Yes! Learn more.' ) }
 					</p>
 
 					<h2>
-						{ __( 'Speed up your site and its content' ) }
+						{ __( 'Build your Jetpack site with blocks' ) }
 					</h2>
 
 					<p>
-						{ __( 'Jetpack 6.7 introduces a new feature to help make your site faster. Our site accelerator already ' +
-							'offered speedier and optimized images served from our global Content Delivery Network. ' +
-							'Now we can also speed up your site by serving your static files (think CSS and JavaScript) from the same network.'
-						) }
-					</p>
-
-					<p>
-						{ __( 'Turn on one or both and see decreased page load speeds, ' +
-						'as well as reduced bandwidth usage—which may lead to lower hosting costs.'
+						{ __( 'Today, we are introducing the first wave of Jetpack-specific blocks built specifically ' +
+							'for the new editor experience: payment buttons, contact forms, maps, and markdown.'
 						) }
 					</p>
-
 					<p>
-						{ __( 'Enable site acceleration now by clicking the toggle below.' ) }
+						<img
+							src={ imagePath + 'block-picker.png' }
+							width="250"
+							alt={ __( 'Jetpack is ready for the new WordPress editor' ) } />
 					</p>
-
-					<div className="jp-upgrade-notice__enable-module">
-
-						<SettingsGroup hasChild>
-							<CompactFormToggle
-								checked={ siteAcceleratorStatus }
-								toggling={ togglingSiteAccelerator }
-								onChange={ this.handleSiteAcceleratorChange }
-								disabled={ ! canDisplaySiteAcceleratorSettings }
-							>
-								<span className="jp-form-toggle-explanation">
-									{ __( 'Enable site accelerator' ) }
-								</span>
-							</CompactFormToggle>
-							<FormFieldset>
-								<ModuleToggle
-									slug="photon"
-									disabled={ this.props.isUnavailableInDevMode( 'photon' ) }
-									activated={ this.props.getOptionValue( 'photon' ) }
-									toggling={ this.props.isSavingAnyOption( 'photon' ) }
-									toggleModule={ this.toggleModule }
-								>
-									<span className="jp-form-toggle-explanation">
-										{ __( 'Speed up image load times' ) }
-									</span>
-								</ModuleToggle>
-								<ModuleToggle
-									slug="photon-cdn"
-									disabled={ false }
-									activated={ this.props.getOptionValue( 'photon-cdn' ) }
-									toggling={ this.props.isSavingAnyOption( 'photon-cdn' ) }
-									toggleModule={ this.toggleModule }
-								>
-									<span className="jp-form-toggle-explanation">
-										{ __( 'Speed up static file load times' ) }
-									</span>
-								</ModuleToggle>
-							</FormFieldset>
-						</SettingsGroup>
-					</div>
-
 					<div className="jp-dialogue__cta-container">
 						<Button
```

Comment:
I think you will need to update this button as well; it currently links to the site accelerator doc.

Reply:
We could maybe link to our release post here?
https://wp.me/p1moTy-cee

Reply:
Since the link text was updated to "Take me to the new editor", I updated the link to point to the post editor in d0cbb3f09cb36dc5eeb9fc441e0ccb7028e59fec